mvp-spec/publishers-requirements.md

```diff
@@ -1,26 +1,23 @@
-# Ad Server Implementation
+# Publishers requirements
 
 ## Goal of the document
 
-This document provides the guidance and the requirements for implementing 
-Prebid Addressability Framework (PAF) on an Ad Server.
+This document describes what is required for a Publisher to participate in the Prebid Addressability Framework (PAF).
 
 ## Overview
 
-For performance purposes, it is recommended to implement the 
-Addressable Content Service directly in the Ad Server. Therefore, the following
-document is focused on this setup.
+The Prebid Addressability Framework enhances ad slots selling by instantiating an audit trail. This audit trail requires the following elements, that Publishers must implement:
+* a Seed, identifying an bid request for an ad slot for a given user
+* Transmission Requests, identifying the Seed and the requester at each step of the bidding workflow
+* Transmission Responses, as answers to Transmission Requests
+* an Audit Log, identifying which entities have been involved in the display of an ad
 
-Compared to a usual setup with an Ad Server, the Publisher or the Provider of
-the Ad Server has to implement additional features to enable PAF. 
+Seeds, Transmission Requests, and Transmissions Responses are signed.
 
-Those features are:
-* The generation of the Seeds of the Addressable Contents;
-* The emission of Transmissions to the Ad Network;
-* The support of the Audit Log for the user;
-* The exposition of an Identity endpoint.
+Publishers are also required to:
+* Expose an Identity endpoint, so that participants can verify the the publisher's signature.
 
-The following diagram introduces an overview of this setup:
+Process overview:
 
 <!--partial-begin { "files": [ "ad-server-flow.mmd" ], "block": "mermaid" } -->
```

Comment:
Shouldn't this be removed? (it seems redundant with what's in the workflows.md file)

Reply:
Agreed, updated.